packages/web/config/ibc-assets.ts

```diff
@@ -39,10 +39,6 @@ export const IBCAssetInfos: (IBCAsset & {
       method: "deposit-address" as const,
       sourceChains: [
         AxelarSourceChainConfigs.usdc.ethereum,
-        AxelarSourceChainConfigs.usdc.bnbChain,
-        AxelarSourceChainConfigs.usdc.avalanche,
-        AxelarSourceChainConfigs.usdc.polygon,
-        AxelarSourceChainConfigs.usdc.fantom,
         AxelarSourceChainConfigs.usdc.moonbeam,
```

Comment:
Lets remove moonbeam for now as well.  There's almost no Axelar USDC on moonbeam